### PR TITLE
Fix slash to backslash conversions in url hashes for opening links in incognito

### PR DESCRIPTION
**Pull request by askepticaldreamer**
…nito links

Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Bugfix for https://github.com/Chatterino/chatterino2/issues/4279

Link was being passed to startDetached function as part of the command string. I suspect somehow this is what is causing / to be changed to \ on windows. Passed the link as part of the arguments and the issue went away. Tested on chrome and firefox.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 - Bugfix: Fixed crash that could occur when closing down a split at the wrong time. (#4277)
 - Bugfix: Fixed crash that could occur when closing down the last of a channel when reloading emotes. (#4278)
 - Bugfix: Fixed scrollbar highlight colors when changing message history limit. (#4288)
+- Bugfix: Fixed slashes getting changed to backslashes in url hash when opening links in incognito on windows (#4307)
```
Comment (Felanbird) [inline CHANGELOG.md:26]:
```suggestion
- Bugfix: Fixed an issue on Windows when opening links in incognito mode that contained forward slashes. (#4307)
```
Better user-facing changelog

File: src/util/IncognitoBrowser.cpp [hunk 2]
```diff
@@ -100,7 +97,7 @@ bool openLinkIncognito(const QString &link)
     auto command = getCommand(link);
```
Comment (kornes) [inline src/util/IncognitoBrowser.cpp:97]: remove now unused argument for `getCommand()`

**Review by Felanbird — COMMENTED** [review]
Functionality wise works as expected on Windows 👍 

only tested firefox but I'm doubtful that matters

**Review by Felanbird — APPROVED** [review]

**Review by pajlada — APPROVED** [review]

**Comment by Felanbird** [comment]
@askepticaldreamer Thanks for your contribution! As a ~~first~~ second-time contributor, you can now add yourself to the contributors list that's shown inside the application.

To do so, open a new PR where you modify the resources/contributors.txt file and add yourself to the list. Make sure to read the comments at a top for instructions.

**Pull request merged**